internal/key/key.go

```diff
@@ -95,21 +95,6 @@ func GenerateAssetName(values ...string) string {
 	return strings.Join(values, "-")
 }
 
-// IsCAPIVersion returns whether a given GS Release Version uses the CAPI projects
-func IsCAPIVersion(version string) (bool, error) {
-	capiVersion, err := semver.New(FirstCAPIRelease)
```

Comment:
should we also remove `FirstCAPIRelease` const?

Reply:
done